packages/contracts-bedrock/slither-report.json

```diff
@@ -0,0 +1,1177 @@
+{
```

Comment:
Can we format this as file as an array of these objects? Right now these findings can't be parsed